compiler/rustc_mir_transform/src/coroutine/by_move_body.rs

```diff
@@ -219,6 +219,8 @@ pub(crate) fn coroutine_by_move_body_def_id<'tcx>(
         mir::MirSource::from_instance(InstanceKind::Item(body_def.def_id().to_def_id()));
     dump_mir(tcx, false, "built", &"after", &by_move_body, |_, _| Ok(()));
 
+    // Feed HIR because we try to access this body's attrs in the inliner.
+    body_def.feed_hir();
```

Comment:
`feed_hir` both feeds the `local_def_id_to_hir_id` query, and also the HIR attrs (as an empty list), which is fine here b/c you can't put attrs on the child coroutine anyways.